src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -53,11 +54,15 @@ public function process(ContainerBuilder $container)
         $typeExtensions = array();
 
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
-            $alias = isset($tag[0]['alias'])
-                ? $tag[0]['alias']
-                : $serviceId;
+            $extendedType = $serviceId;
```

Comment:
we need a deprecation to be triggered when the service id is used, because such case makes no sense actually.